### PR TITLE
Update deps 21 and nuget versioning

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: build_projects/update-dependencies/update-dependencies.csproj [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.VersionTools" Version="$(VersionToolsVersion)" />
+    <PackageReference Include="Nuget.Versioning" Version="4.5.0" />
+    <PackageReference Include="Nuget.Packaging.Core" Version="4.5.0" />
```
Comment (dagood) [inline build_projects/update-dependencies/update-dependencies.csproj]: Consider making a dependency property for this and combining with the ref added in https://github.com/dotnet/cli/commit/a4040c9983928d4aef2f82842e16dbb6aa7b8948?

**Comment by mmitche** [comment]
@livarcocc What's with all these test failures?  I've seen them in a few PRs

**Comment by livarcocc** [comment]
@mmitche This failure seems legit. The ASp.NET tools are targeting a 2.1.3 version and the CLI itself has a 2.1.5-servicing version.

The host will not rollforward to a pre-release version, causing the tool to fail to run.

**Comment by mmitche** [comment]
@livarcocc What's the resolution then?

**Comment by livarcocc** [comment]
Not needed anymore.

**Pull request closed without merging**